moto/secretsmanager/models.py

```diff
@@ -565,6 +565,19 @@ def restore_secret(self, secret_id):
 
         return secret.arn, secret.name
 
+    def update_secret_tags(self, secret_id, tags):
+
+        if secret_id not in self.secrets.keys():
+            raise SecretNotFoundException()
+
+        secret = self.secrets[secret_id]
+        old_tags = secret["tags"]
```

Comment:
This is an instance of `FakeSecret`, so you can access the tags by attribute (secret.tags) 
(Tests are currently failing because of this)